server/models/device_feature_state.js

```diff
@@ -20,7 +20,15 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.DOUBLE,
       },
     },
-    {},
+    {
+      indexes: [
+        {
+          name: 'device_feature_id_created_at_index',
+          using: 'BTREE',
+          fields: ['device_feature_id', 'created_at'],
```

Comment:
This won't change anything, to create indexes you need to add them to the migration. I don't think this change is needed

Reply:
I remove it